spec/01-unit/003-prefix_handler_spec.lua

```diff
@@ -54,282 +54,6 @@ describe("NGINX conf compiler", function()
     end)
   end)
 
-  describe("compile_kong_conf()", function()
```

Comment:
I think those tests are still very much needed? We are only removing the integration tests around `kong compile`, but not changing the internals of how we build the Kong prefix directory (still the same way with `kong prepare`).

Reply:
... No idea why I nuked this. whups :p